underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java

```diff
@@ -307,15 +307,18 @@ protected boolean deleteObject(String key) throws IOException {
 
   @Override
   protected List<String> deleteObjects(List<String> keys) throws IOException {
+    if (!Configuration.getBoolean(PropertyKey.UNDERFS_S3A_BULK_DELETE_ENABLED)) {
+      return super.deleteObjects(keys);
+    }
     Preconditions.checkArgument(keys != null && keys.size() <= getListingChunkLengthMax());
     try {
+      List<String> deletedObjects = new ArrayList<>();
```

Comment:
why move this line?

Reply:
It was moved to be used by the fallback logic. Now we reuse the super class logic I will move it back